docs/glossary.rst

```diff
@@ -10,26 +10,25 @@ Astropy Glossary
       A parenthesized number followed by a comma denotes a tuple with one
       element. The trailing comma distinguishes a one-element tuple from a
       parenthesized ``n``.
-      This is from NumPy; see https://numpy.org/doc/stable/glossary.html.
+      This is from NumPy; see https://numpy.org/doc/stable/glossary.html#term-n.
 
    number
-      Any numeric type. eg float or int or any of the ``numpy.number``.
+      Any numeric type. e.g. `float` or `int` or ``numpy.number``.
```

Comment:
shouldn't we use double backticks in rst ?
```suggestion
Any numeric type. e.g. ``float`` or ``int`` or ``numpy.number``.
```

Reply:
Those are automatically resolved as links to the definitions in the Python docs.